<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>238</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>501</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Face Regconition App</string>
  </property>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>441</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>10</y>
     <width>440</width>
     <height>420</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="preButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
